src/components/AuthMethodButton/AuthMethodButton.tsx

```diff
@@ -58,9 +61,16 @@ export default function AuthMethodButton(props: AuthMethodButtonProps) {
   useEffect(() => {
     if (props.href) return;
 
+		let loginHint : string | undefined = undefined;
```

Comment:
Indent-level seems to be off, or is that just GH diff view leading me astray ?

Reply:
No, that is definitely off, hmm.